README.md

```diff
@@ -1,135 +1,450 @@
 
--   At least 6 CPU cores and 12GB of RAM that are available to docker.
+The `solidity` project includes the following files to configure the Chainweb EVM development network:
 
-## Quick Start
+- The `solidity/devnet-accounts.json` file contains all of the account information generated from a test BIP-44 wallet using a seed entropy value of `0x0000 0000 0000 0000 0000 0000 0000 0000` (16 zero bytes).
+- The `solidity/hardhat.config.js` file reads the account information from the `solidity/devnet-accounts.json` file, sets the properties that describe the Chainweb EVM nodes, and configures the `kadena_devnet0` network as the default Hardhat network for the `solidity` project. 
 
-Clone the the Github repository of [this
-project](https://github.com/kadena-io/kadena-evm-sandbox):
+After a project is configured to use the Chainweb EVM development network settings and accounts, you can run unit tests for the project using the standard `hardhat test` command.
+For example:
 
 ```sh
-git clone https://github.com/kadena-io/kadena-evm-sandbox
-cd kadena-evm-sandbox
+cd solidity 
+npx hardhat test
 ```
 
-When running for the first time it is recommended to pull all container images
-before starting the network:
+### Modifying the Hardhat project
+
+To integrate with the Chainweb EVM development network:
+
+1. Copy the `solidity/devnet-accounts.json` file into the root directory of your Hardhat project. 
+2. Open the `hardhat.config.js` file for your Hardhat project in your code editor. 
+3. Copy and paste the code to read account information from the `solidity/hardhat.config.js` file into the `hardhat.config.js` file for your project.
+   
+   For example:
+
+   ```javascript
+   require("@nomicfoundation/hardhat-toolbox");
+   require("@nomicfoundation/hardhat-verify");
+   const path = require("path");
+   const fs = require("fs");
+   
+   // Read and parse the accounts file
+   const devnetAccountsPath = path.join(__dirname, 'devnet-accounts.json');
+   const devnetAccountsFile = fs.readFileSync(devnetAccountsPath, 'utf8');
+   const devnetAccounts = JSON.parse(devnetAccountsFile);
+   
+   // Validate account configuration
+   const requiredAccounts = 20;
+   if (devnetAccounts.accounts.length !== requiredAccounts) {
+     throw new Error(`Expected ${requiredAccounts} accounts in devnet-accounts.json, found ${devnetAccounts.accounts.length}`);
+   };
+   ```
+
+4. Copy and paste the code to configure network information from the `solidity/hardhat.config.js` file into the `hardhat.config.js` file for your project. 
+
+   For example:
+
+   ```javascript
+     defaultNetwork: "kadena_devnet0",
+     networks: {
+       kadena_devnet0: {
+         url: 'http://localhost:8545',
+         chainId: 1789,
+         accounts: devnetAccounts.accounts.map(account => account.privateKey),
+         chainwebChainId: 0,
+       },
+       kadena_devnet1: {
+         // url: 'http://localhost:8546',
```

Comment:
This should be removed. I removed it from the actual file and pushed the change.

Reply:
The demo is not set up to use the plugin. John said multiple times he wants the chainweb evm to be demoed at ETHDenver. If we configure the hardhat plugin, we are not demoing chainweb EVM.

Reply:
>@hswopeams Can you clarify what "This" refers to? This step? This line 274? This javascript snippet?
The commented line should be removed from the README. I already removed it from the actual config file.